app/house.py

```diff
@@ -162,6 +162,46 @@ def all_user_houses():
     )
 
 
+@house_blueprint.route("/<int:house_id>/task/all")
+@jwt_required
+def get_house_tasks(house_id: int):
```

Comment:
I think you don't need to force `int` if you've already done it in the route (`<int:house_id>`). Like we've been doing on the other cases

Reply:
Probably not.
```suggestion
def get_house_tasks(house_id):
```